src/Common/src/Microsoft/Win32/SafeHandles/SafeX509Handles.Unix.cs

```diff
@@ -92,4 +92,26 @@ public override bool IsInvalid
             get { return handle == IntPtr.Zero; }
         }
     }
+
+    [SecurityCritical]
+    internal sealed class SafeSharedX509StackHandle : SafeHandle
+    {
+        internal static readonly SafeSharedX509StackHandle InvalidHandle = new SafeSharedX509StackHandle();
+
+        private SafeSharedX509StackHandle() :
+            base(IntPtr.Zero, ownsHandle: false)
+        {
+        }
+
+        protected override bool ReleaseHandle()
+        {
+            SetHandle(IntPtr.Zero);
```

Comment:
I found an example in [SafeCredentialReference](http://referencesource.microsoft.com/#Syste/net/System/Net/_SafeNetHandles.cs,1310); then added some `Interlocked.Increment` and `Interlocked.Decrement` and `Console.WriteLine`s to let me test this out:

```
Starting:    System.Security.Cryptography.X509Certificates.Tests
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
There is/are 1 active SafePkcs7Handle(s)
There is/are 0 remaining SafePkcs7Handle(s)
Finished:    System.Security.Cryptography.X509Certificates.Tests
```

So things work the way that I expected them to.  Thanks @jkotas.